tests/profiles/test_base_profile.py

```diff
@@ -47,6 +48,75 @@ def test_disable_event_tracking(disable_event_tracking: str):
     )
     profile_contents = yaml.safe_load(test_profile.get_profile_file_contents(profile_name="fake-profile-name"))
 
-    assert ("config" in profile_contents) == disable_event_tracking
     if disable_event_tracking:
-        assert profile_contents["config"]["send_anonymous_usage_stats"] is False
+        assert profile_contents["config"]["send_anonymous_usage_stats"] == "False"
+
+
+def test_disable_event_tracking_and_send_anonymous_usage_stats():
+    expected_cosmos_error = (
+        "Cannot set both disable_event_tracking and "
+        "dbt_config_vars=DbtProfileConfigVars(send_anonymous_usage_stats ..."
+    )
```

Comment:
This expected error message is not checked in the test assertion below. You can use `with pytest.raises(CosmosValueError, match='regex match here')` to check that the raised error message matches a regex pattern.

Reply:
fix)